python/proton/_wrapper.py

```diff
@@ -135,5 +141,7 @@ def __repr__(self):
                                         id(self), addressof(self._impl))
 
 
-PYCTX = int(pn_py2void(Wrapper))
+# PYCTX = int(pn_py2void(Wrapper))
+print(f"log _wrapper line 144 : type {type(Wrapper)} \n wrapper {Wrapper}")
+PYCTX = int(Wrapper)
```

Comment:
My first thought would be to do `PYCTX = ffi.cast('void *', Wrapper)` here. I'll have a closer look in the evening and try it out.

The intent of this code seems to me to keep around the pointer and for some reason it is represented on the Python side as an int. No reason for that, IMO. Keep it be a void pointer.

Reply:
Uh, disregard that. Hmm, this is in the realm of callbacks and things like that. https://cffi.readthedocs.io/en/latest/using.html#extern-python-new-style-callbacks I'll need to understand the use and look into this more; figure out how to accomplish this with CFFI.